docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+
+## Prometheus as software
+
+Prometheus isn’t just a data format; many people refer to it as an [open source systems monitoring and alerting toolkit](https://prometheus.io/docs/introduction/overview/). That’s because Prometheus is software, not just data.
+
+Prometheus can scrape metric data from software and infrastructure and store it. Scraping means that Prometheus software periodically revisits the same endpoint to check for new data. Prometheus “scrapes” data from a piece of software that is instrumented with a client library.
+
+For example, a NodeJS application can configure the [prom-client](https://github.com/siimon/prom-client) to expose metrics easily at an endpoint, and Prometheus can regularly scrape that endpoint. There are a number of other tools available within the Prometheus toolkit that you can use to instrument applications.
+
+## Prometheus as deployment
+
+The first section of this document introduced the _Prometheus as Data_ concept and how the Prometheus data model and metrics are organized. The second section introduced the concept of _Prometheus as Software_ that is used to collect, process, and store metrics.
+
+This section describes how “Prometheus as Data" and “Prometheus as Software” come together. Suppose an application called `MyApp` uses a Prometheus client to expose metrics. One approach to collecting metrics data is to use a URL in the application that points to an endpoint `http://localhost:3000/metrics` that produces Prometheus metrics data.
+
+The following image shows the two metrics associated with the endpoint. The HELP text explains what the metric means, and the TYPE text indicates what kind of metric it is (in this case, a gauge). `MyAppnodejs_active_request_total` indicates the number of requests (in this case, `1`). `MyAppnodejs_heap_size_total_bytes` indicates the heap size reported in bytes. There are only two numbers because this data shows the value at the moment the data was fetched.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/endpoint-data.png" max-width="750px" caption="Endpoint example" >}}
+
+These metrics are available in an HTTP endpoint, but how do they get to Grafana, and subsequently, into a dashboard?
+
+That’s where you can use either the Prometheus software, or the [Grafana Agent](/docs/agent/latest/). The Grafana Agent collects and forwards telemetry data to open sourcevdeployments of the Grafana Stack, Grafana Cloud, or Grafana Enterprise, where your data can then be analyzed.
+
+Telemetry refers to the process of recording and transmitting the readings of an application or piece of infrastructure. Telemetry is critical to observability because it helps you understand exactly what's going on in your infrastructure. Telemetry data is a source of truth.
```

Comment:
I think we need to tease this out a bit more - explain telemetry data in the context of metrics data. Because, as a beginner, it seems like we've switched our data focus, which is confusing.

Reply:
Best suggestion I can make here to tie the two together is to add a sentence at the end:  "The metrics data that Prometheus helps us with is only one type of telemetry; the other two kinds are logs and traces, covered in separate articles."

Reply:
Done.